powerline/segments/common/bat.py

```diff
@@ -196,10 +183,11 @@ def battery(pl, format='{capacity:3.0%}', steps=5, gamify=False, full_heart='O',
 	Highlight groups used: ``battery_full`` or ``battery_gradient`` (gradient) or ``battery``, ``battery_empty`` or ``battery_gradient`` (gradient) or ``battery``.
```

Comment:
With my suggestions regarding gamify this is how this line should look like: (just copy it literally)

    Highlight groups used: ``battery_full`` or ``battery_gradient`` (gradient) or ``battery``, ``battery_empty`` or ``battery_gradient`` (gradient) or ``battery``, ``battery_online`` or ``battery_ac_state`` or ``battery_gradient`` (gradient) or ``battery``, ``battery_offline`` or ``battery_ac_state`` or ``battery_gradient`` (gradient) or ``battery``.